app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,102 +1,86 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
+import Item from "components/Common/Item";
 import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
+import EditNotePane from "./Pane/Edit";
 
+import { NOTES_DATA } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import NewNotePane from "./Pane/Create";
-import Table from "./Table";
 
 const Notes = () => {
-  const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedNoteIds, setSelectedNoteIds] = useState([]);
   const [notes, setNotes] = useState([]);
+  const [category, setCategory] = useState("All");
 
   useEffect(() => {
-    fetchNotes();
+    setNotes([...NOTES_DATA]);
   }, []);
 
-  const fetchNotes = async () => {
-    try {
-      setLoading(true);
-      const { data } = await notesApi.fetch();
-      setNotes(data.notes);
-    } catch (error) {
-      logger.error(error);
-    } finally {
-      setLoading(false);
-    }
-  };
-
   if (loading) {
     return <PageLoader />;
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            onClick={() => setShowNewNotePane(true)}
-            label="Add New Note"
-            icon="ri-add-line"
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
+    <>
+      <Menubar
+        category={category}
+        setCategory={setCategory}
       />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                label="Delete"
-                icon={Delete}
-                onClick={() => setShowDeleteAlert(true)}
-                disabled={!selectedNoteIds.length}
-              />
-            }
-          />
-          <Table
-            setSelectedNoteIds={setSelectedNoteIds}
-            notes={notes}
-            fetchNotes={fetchNotes}
+      <Container>
+        <Header
+          title="Notes"
+          actionBlock={
+            <Button
+              onClick={() => setShowNewNotePane(true)}
+              label="Add New Note"
+              icon="ri-add-line"
+            />
+          }
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
+          menuBarToggle
+        />
+        {notes.length ? (
+          notes?.map((note, key) => <Item key={key} note={note} />)
```

Comment:
See if `note` has a better unique identifier. If yes, use it as the key. Also here naming the arrow functions arg as `key` is not right. It should've been named `index`.

https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9